batch-jberet/pom.xml

```diff
@@ -75,6 +75,17 @@
             <groupId>org.jboss.msc</groupId>
             <artifactId>jboss-msc</artifactId>
         </dependency>
+        <dependency>
+            <groupId>org.kohsuke.metainf-services</groupId>
+            <artifactId>metainf-services</artifactId>
+            <scope>provided</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.picketbox</groupId>
+            <artifactId>picketbox</artifactId>
```

Comment:
No new PicketBox dependencies can be added to the server.

Reply:
Ouch, that was wrongly resolved merge conflict. Should be fixed now.